pkg/ctl/cmdutils/configfile.go

```diff
@@ -45,6 +46,8 @@ var (
 	)
 )
 
+var multiErr *multierror.Error
```

Comment:
I think it's best `not` to have package variable, which might cause side effect. This also makes unit test harder as you need to clear the value of this multiErr before any test.

Reply:
Any suggestions as to where to declare multiErr?  At the function level for each function? Or in the `commonClusterConfigLoader` struct?  I would think to avoid any side effects, isolating a multierror instance to each function would be best.  Hopefully I am making logical sense, as I am new to this.  I got help on how to debug go tests, what go testing framework is being used etc. so I have a place to go from here.

Reply:
I feel like if it could be local function variable for public method, and argument for private method (so that you can consolidate all checks :+1: )